crates/pathfinder/src/state/block_hash.rs

```diff
@@ -1097,4 +1104,633 @@ mod tests {
         ));
         assert_eq!(compute_final_hash(&header).unwrap(), expected_hash);
     }
+
```

Comment:
Shouldn't we add these as fixtures to gateway-client-fixtures instead? It would make the test much more readable.

Reply:
Oh definitely, thanks for noticing this :bow: